### PR TITLE
Added My Journey Blog Post

### PR DESCRIPTION
**Pull request by nitishkumar71**
Signed-off-by: Nitishkumar Singh <nitishkumarsingh71@gmail.com>

## Description

Blog post about my journey in the OpenFaas community
## Motivation and Context
Share details about how I started contribution in project. 

## Have you applied the editorial and style guide to your post?

See the [README.md](README.md)

## How have you tested the instructions for any tutorial steps?
Yes. The commands were used in bash terminal

## Types of changes

- [x] New blog post
- [ ] Updating an existing blog post
- [ ] Updating part of an existing page
- [ ] Adding a new page

## Checklist:

- [x] I have given attribution for any images I have used and have permission to use them under Copyright law
- [x] My code follows the [writing-style of the publication](README.md) and I have checked this
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: _posts/2022-01-30-my-journey-in-openfaas-community.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+image: /images/kubernetes-operator-crd/pexels-asphalt-blue-sky-clouds-490411.jpg
+categories:
+  - faasd
+  - certifier
+  - openfaas
+author_staff_member: nitishkumar
+dark_background: true
+---
+
+## How I Came to Know about OpenFaas
```
Comment (alexellis) [inline _posts/2022-01-30-my-journey-in-openfaas-community.md:14]: OpenFaas => OpenFaaS :slightly_smiling_face:

**Comment by alexellis** [comment]
> How have you tested the instructions for any tutorial steps?

Could you run the compose locally and do a screenshot, check that the layout looks as expected and the image etc.

File: _posts/2022-01-30-my-journey-in-openfaas-community.md (hunk 1 above)
Comment (nitishkumar71) [reply _posts/2022-01-30-my-journey-in-openfaas-community.md]: Done

**Comment by nitishkumar71** [comment]
> > How have you tested the instructions for any tutorial steps?
> 
> Could you run the compose locally and do a screenshot, check that the layout looks as expected and the image etc.

Thank you for suggestion. It looks ok in terms of layout and image.

File: _posts/2022-01-30-my-journey-in-openfaas-community.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: My Journey in OpenFaaS Community So Far
```
Comment (alexellis) [inline _posts/2022-01-30-my-journey-in-openfaas-community.md:2]: How about? "My Journey Contributing To The OpenFaaS Community So Far"

File: _posts/2022-01-30-my-journey-in-openfaas-community.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+description: "Nitishkumar shares his experience of working in the OpenFaaS community"
```
Comment (alexellis) [inline _posts/2022-01-30-my-journey-in-openfaas-community.md:3]: "Nitishkumar shares his experiences from contributing code to OpenFaaS and joining the community"

File: _posts/2022-01-30-my-journey-in-openfaas-community.md (hunk 2 above)
Comment (nitishkumar71) [reply _posts/2022-01-30-my-journey-in-openfaas-community.md]: Thank You for suggestion. Changes are done.

File: _posts/2022-01-30-my-journey-in-openfaas-community.md (hunk 3 above)
Comment (nitishkumar71) [reply _posts/2022-01-30-my-journey-in-openfaas-community.md]: Thank You for suggestion. Changes are done.

**Comment by alexellis** [comment]
Have you done a local preview yet by running the site on your machine?

Please could you and check that it all looks like you're expecting?

**Pull request closed without merging**